api/src/main/proto/spine/protodata/source.proto

```diff
@@ -46,11 +46,12 @@ message ProtobufSourceFile {
 
     // The relative path to the file.
     //
-    // Must be the same as `file.path`.
+    // Must be the same as `header.file`.
     //
-    FilePath file_path = 1;
+    File file = 1;
 
-    File file = 2;
+    // A header of the Protobuf file.
```

Comment:
Let's append this notion of the equality of `file` and `header.file`, we do in another file in this changeset.